### DISCUSSION TITLE
[Nemo] Changelog for 6.0.0

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [linuxmint/discussions#250] [Development] [answered]
I see that version 6.0.0 of nemo [was tagged around a month ago](https://github.com/linuxmint/nemo/releases/tag/6.0.0).

Are there any release notes or a changelog for this version? I want to upgrade the Alpine package for nemo, but want to be aware of any breaking changes (TBH, I'm also just curious as to what's changed such that the major version was bumped).

Looks like [the changelog in the repository](https://github.com/linuxmint/nemo/blob/master/ChangeLog) is an autogenerated file that was last updated in 2012.

**Comment by rcalixte** [comment, marked as answer]
The Nemo changelog is located here: https://github.com/linuxmint/nemo/blob/master/debian/changelog. The versioning bumps every release so the next version will be 6.2.x and so on. This is similar to the kernel versioning except that it is less arbitrary. So the major version changing is not indicative of any breaking changes, just that 6.0 comes after 5.8 in the release ordering.

This changelog includes everything from 1.0.0 so I think it is fair to call this the canonical changelog for Nemo. The changelog you linked looks like it's from Nautilus at the time of the fork.

**Reply by WhyNotHugo** [reply]
Thanks
